### PR TITLE
Update our laptop script and add manual installation instructions.

### PR DESCRIPTION
**Pull request by johnpaulashenfelter**
We haven't updated the laptop script from thoughtbot (https://github.com/thoughtbot/laptop) in quite some time. Now we have!

I also added some manual setup steps that leverage `brew bundle` to setup
the absolute bare minimal of Unix tools and add the key OSX apps that are
available through `brew cask`. Now everyone, including me, can me happy
setting up a new machine without _too much_ effort.

### CODE REVIEW

**Comment by tute** [comment]
I'm surprised CI is marked as failed if there are no tests to run?

Skimmed the update and looks good to me. I use thoughtbot/laptop myself with some customizations ([`tute/dotfiles/laptop.local`](https://github.com/tute/dotfiles/blob/master/laptop.local)), so I didn't test it though.

**Comment by johnpaulashenfelter** [comment]
@tute I disabled the plugin and removed it from circle. Going to try an empty commit to clear it all up.

**Comment by johnpaulashenfelter** [comment]
Boom.

**Comment by tute** [comment]
Checked the differences with latest thoughtbot's (which works well for me) and found two no yarn, Heroku tools, or OpenSSL. If we don't need them at Epion, 🚢!

File: Brewfile.unix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+brew "tmux"
+
+brew "hub"
+
+brew "imagemagick@6"
+
+brew "qt@5.5"
+
+brew "node"
+brew "rbenv"
```
Comment (mattracki) [inline Brewfile.unix]: So rbenv will be the default going forward.  @sebarmano & @amygimma are you OK with this? 

I just installed on my machine, and that was the only conflict I ran into, from our old config.

**Comment by mattracki** [comment]
@EpionHealth/epion-dev all OK with switching to rbenv as default.  Good work!

**Pull request merged**